keyboards/herevoland/buff75/info.json

```diff
@@ -0,0 +1,139 @@
+{
+    "keyboard_name": "Buff75",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "manufacturer": "HereVoLand",
+    "url": "",
+    "maintainer": "Here VoLand @Vem",
+    "usb": {
+        "vid": "0xB727",
+        "pid": "0xB727",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 1,
+        "brightness_steps": 8,
+        "saturation_steps": 8,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    }, 
+    "ws2812": {
+        "pin": "B2"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "A8", "B15", "B14", "B13", "B12", "A3", "A4" ],
+        "rows": ["A5", "B10", "B1", "B0", "A7", "A6"]
+    },
+    "layouts": {
+        "LAYOUT": { 
```

Comment:
Providing a single layout macro, when your matrix_diagram shows multiple is not correct.

Reply:
Just deleting the `matrix_diagram.md` file does not exclude you from making these changes.

Your product photo clearly shows multiple available layouts.

Reply:
Yes, we considered canceling the solder version, but we just forgot to replace the product photo

Reply:
So if a PR to the-via/keyboards gets added with multiple layout support, you agree we can remove this code from the repo due to it being raised under false pretences?

Reply:
Agreed, this is reasonable.